### ISSUE TITLE
Build Quality checks - show required threshold in report 

### ISSUE THREAD

**Issue by melezhik** [MicrosoftPremier/VstsExtensions#209]
Right now the task only shows current code coverage and if this passes the required one:

```
Using IdentifierJobResolver
Validating code coverage policy...
Successfully read code coverage data from build.
Evaluating coverage data from 1 filtered code coverage data sets...
Total lines: 166
Covered lines: 158
Code Coverage (%): 95.1807
[SUCCESS] Code coverage policy passed with 95.1807% (158/166 lines).
``` 

I would like that the tasks also prints out the required threshold, something like that ( see `Required Code Coverage` line ):

```
Code Coverage (%): 95.1807
Required Code Coverage (%): 90
[SUCCESS] Code coverage policy passed with 95.1807% (158/166 lines).
```

**Comment by ReneSchumacher** [comment]
Hi @melezhik,

thanks for the suggestion, we will add the threshold information to the log output in the next version of the task.

**Comment by ReneSchumacher** [comment]
Hi  @melezhik,

I'm closing this issue now as it has been implemented in the latest version (v9.0.0), which should be released today.

Thanks for your feedback!
